packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -728,6 +728,14 @@ new stepfunctions.Parallel(this, 'All jobs')
     .branch(new MyJob(this, 'Slow', { jobFlavor: 'slow' }).prefixStates());
 ```
 
+To retrieve all possible subsequent states from an initial state, you can use the
+method `State.findReachableStates`:
+
+```typescript
+        import * as sf from '@aws-cdk/aws-stepfunctions';
+        const nextState = sf.State.findReachableStates(startState); // All reachable / next states
+```
+
```

Comment:
I don't think you need the code block for this since the API is straightforward. How about changing it to something like this -

```suggestion
A few utility functions are available to parse state machine fragments.
* `State.findReachableStates`: Retrieve the list of states reachable from a given state.
* `State.findReachableEndStates`: Retrieve the list of end or terminal states reachable from a given state.
```

Reply:
Dona